### ISSUE TITLE
Blender Crash with latest Build

### ISSUE THREAD

**Issue by J-J-Cyber** [LuxCoreRender/LuxCore#280]
Blender 2.81a
LuxRender-Latest Build

Python argument types in
    Scene.DefineBlenderMesh(Scene, str, int, int, int, int, int, list, list, int, list)
did not match C++ signature:
    DefineBlenderMesh(class luxcore::detail::SceneImpl * __ptr64, class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >, unsigned __int64, unsigned __int64, unsigned __int64, unsigned __int64, unsigned __int64, unsigned __int64, unsigned __int64, unsigned int)
    DefineBlenderMesh(class luxcore::detail::SceneImpl * __ptr64, class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >, unsigned __int64, unsigned __int64, unsigned __int64, unsigned __int64, unsigned __int64, unsigned __int64, unsigned __int64, unsigned int, class boost::python::api::object)

**Comment by Dade916** [comment]
It looks like you have an installation with a mismatch between BlendLuxCore and LuxCore binary versions.

You should delete the installation, re-download the plugin and re-install. It should fix your problem.

**Comment by J-J-Cyber** [comment]
Hi,

Have deleted Blender 2.81a, resinstalled it and downloaded the latest
version. Now I get the following error:

**Comment by J-J-Cyber** [comment]
I've deleted now also the %AppData%-Folder from Blender, now the
Blender-Add-on shows no error anymore. However, By trying out the
example files, blender crashes. Sometimes:

  * without error message
  * with clBuilder-error

maybe it is because of my computer setup. I've borrowed a graphic card
which has OpenCL support and another which has NO OpenCL support. I
think that that is the reason for all those crashes. There are two cards...
